odml/property.py

```diff
@@ -373,45 +373,73 @@ def clone(self):
 
         return obj
 
-    def merge(self, other, strict=True):
+    def merge_check(self, source, strict):
```

Comment:
do we want to make them "private" or should they be part of the public library?  ```__merge_check```

Reply:
Single underscore :)

Reply:
I thought so too at the beginning, but the `Property.merge_check()`  is accessed externally when doing `Section` merges, therefore they should stay public.

Reply:
Might also be useful for the user, at the end of the day.